README.md

```diff
@@ -185,8 +189,8 @@ gcloud container clusters get-credentials --zone=$LOCATION $CLUSTER
 You can then apply the Jetstack Secure agent configuration to your cluster:
 
 ```sh
-kubectl -n $NAMESPACE apply  -f agent-config.yaml
-kubectl -n $NAMESPACE rollout restart deploy jetstack-secure-preflight
+cat agent-config.yaml | sed '/namespace:/d' | kubectl -n $NAMESPACE apply -f-
+kubectl -n $NAMESPACE rollout restart $(kubectl -n $NAMESPACE get deploy -oname | grep preflight)
```

Comment:
I remember about having to separate behaviours:
- When running the smoke-test, we set replica=0
- When running normally (e.g., one click deployment through the marketplace UI), the preflight deployment would be set to replica=1 (= purposefully failing)

I don't really like the idea of letting the preflight deployment in a failing mode "by default", e.g. the user would immediately see a failing deployment after deploying:

<img src="https://user-images.githubusercontent.com/2195781/110791519-9acde700-8272-11eb-81f4-4f27fb8a174d.png" width="300" alt="The test-1 application page on GKE should show the test-1 application. The preflight deployment is failing because the user has not (yet) gone to http://platform.jetstack.io/ to register their cluster. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">

Reply:
I'll open an issue about this.